examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+# Deferred Processing and Transactions
+
+## Introduction
+[Deferred processing](https://docs.pachyderm.com/latest/how-tos/deferred_processing/) 
+is a Pachyderm technique for controlling when data gets processed.
+Transactions are Pachyderm feature for assuring data from different repos get processed together.
+
+This is an example which uses a simple DAG, 
+based on our opencv example, 
+to illustrate using deferred processing along with transactions.
+
+
```

Comment:
## Prerequisites 

Before you begin, you need to have the following components installed and configured:

- Pachyderm v1.9.8 or later installed on your platform of choice or locally. See [Deploy Pachyderm](link to the deployment section).
- anything else?

If it's just Pachyderm, then get rid of the list:
Before you begin, you need to have Pachyderm v1.9.8 or later installed on your computer or cloud platform. See [Deploy Pachyderm](link to the deployment section).